src/components/add-movie/add-movie.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
```

Comment:
It's okay. JFYI https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html